spec/rubocop/cop/style/method_call_with_args_parentheses_spec.rb

```diff
@@ -404,6 +404,14 @@ def method_missing(name, ...)
       end
     end
 
+    context 'numeric blocks in 2.7', :ruby27 do
```

Comment:
That's my two cents. The canonical name "numbered parameters" instead of "numeric blocks" may be better.

```suggestion
    context 'numbered parameters in 2.7', :ruby27 do
```